rebac-admin-backend/Makefile

```diff
@@ -22,5 +22,9 @@ mocks:
 
 # Run tests
 test: mocks
-	go test ./...
+	go test ./... -cover -coverprofile coverage_source.out
```

Comment:
Since we're using this Makefile target in CI, we should be able to disable caching. The idiomatic way to do this is to call `go test` with `-count=1` option.

In the Makefile, we can do it like this:

```Makefile
test: mocks
    go test ./... -cover -coverprofile coverage_source.out $(ARGS)
    # ...
```

And run it in the CI with:

```
make ARGS="-count=1" test
```

Reply:
Great! I'll add this right away